### ISSUE TITLE
Problem with selecting project in the past

### ISSUE THREAD

**Issue by HeinzWuert** [kevinpapst/kimai2#3657]
### Describe the issue

A project that is now created for an expired month does not appear in the selection list for changing the project name

### I already tried

- [X] I've read and searched [the documentation](https://www.kimai.org/documentation/).
- [X] I've searched for similar issues in this repository.
- [X] I've searched for similar issues in [the discussions](https://github.com/kevinpapst/kimai2/discussions).

### Kimai version

1.29 Demo and 1.8 own

### Which PHP version are you using?

Other (please mention below)

### How do you run Kimai?

Self-Hosted

### Steps to reproduce the behavior

Example
Project is running since Oct up to now
The situation requires subsequent splitting of the project for Oct
A new project will be created with a term of only October
Now the October bookings are to be moved from the old project to the newly created one.
However, the newly created project does not appear as a target in the project selection

**Comment by kevinpapst** [comment]
Yes, you have to change the customer to reload the project list. That was always the case unfortunately.
